cachelib/allocator/CacheAllocator.h

```diff
@@ -944,8 +944,17 @@ class CacheAllocator : public CacheBase {
   //
   // @return  time when the cache was created.
   time_t getCacheCreationTime() const noexcept { return cacheCreationTime_; }
+
+  // unix timestamp when the NVM cache was created. If NVM cahce isn't enaled,
+  // the cache creation time is returned instead.
```

Comment:
Thank you for adding the comment.

Please return 0 here if the nvm cache is not initialized just like before, and modify this comment accordingly. We don't want to change the existing behavior (though arguably your proposed behavior makes more sense) so that we can avoid migrations.

Reply:
Done

Reply:
Done